contracts/access/Whitelist.sol

```diff
@@ -52,8 +52,8 @@ contract Whitelist is Ownable, RBAC {
    * false if all addresses were already in the whitelist
    */
   function addAddressesToWhitelist(address[] _operators)
-    onlyOwner
-    public
+  public
```

Comment:
isn't there an indentation missing here? @elopio you experimented a bit with Solium, any ideas on why it didn't catch this?